### DISCUSSION TITLE
Any future support for TB 115?

### DISCUSSION THREAD

**Discussion by Thalon77** [Aris-t2/CustomCSSforTb#16] [General]
Just a general question now that TB notified me that a total new version (V 115) will be released in july with a total overhauled code and new GUI: will this custom CSS support that?

**Comment by Aris-t2** [comment]
Some things will have to be updated for sure, but in general I plan to supported TB 115+.

**Comment by Aris-t2** [comment]
Just tested 115 Nightly. Using tiny modification to current toolbar button CSS on main toolbar offers this result:

![image](https://github.com/Aris-t2/CustomCSSforTb/assets/6013569/ad6f2859-3881-4c97-839c-806cee6a48fd)

Nothing else tested yet.

**Comment by maryadavies** [comment]
Welp. 115 just came out on the release channel.

Any progress?

**Reply by Aris-t2** [reply]
You can use the test builds for TB 115: https://github.com/Aris-t2/CustomCSSforTb/releases/tag/3.0.0test

**Comment by Aris-t2** [comment]
Current "test3" became [3.0.0 release](https://github.com/Aris-t2/CustomCSSforTb/releases/).

**Reply by maryadavies** [reply]
I'd wondered if it'd gotten forgotten before the release. Thanks.
